jenkins-plugin/pom.xml

```diff
@@ -41,59 +41,34 @@
 
   <properties>
     <hpi.pluginChangelogUrl>https://github.com/jenkinsci/pipeline-maven-plugin/releases</hpi.pluginChangelogUrl>
+    <hpi.compatibleSinceVersion>1332</hpi.compatibleSinceVersion>
```

Comment:
if I understand well https://www.jenkins.io/doc/developer/plugin-development/mark-a-plugin-incompatible/#recording-oldest-compatible-version, should be 1333 no ?

> if your new version is not configuration-compatible with any previous versions, compatibleSinceVersion should use the next release’s version number

Maybe just not updated after rebase ?

Reply:
yes as a release has been made before this get merged. so it's a manual update.
done here 770360f56f24d305300f96abadd6679cab4072f9